configs/constant_presets/minimal.yaml

```diff
@@ -0,0 +1,127 @@
+# Minimal preset
+
+
+# Misc
+# ---------------------------------------------------------------
+
+
+# Just 8 shards in the minimal testing setup
+SHARD_COUNT: 8
```

Comment:
Let's have this file at least mirror the changes here -  https://github.com/ethereum/eth2.0-specs/blob/199e7849dafc48145dfbc6779ef2ccfa04e4a718/tests/phase0/conftest.py#L12-L22

Reply:
Fixed, see c5d2696